### PR TITLE
feat: change proxy target to localhost when test

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
As title.

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
Ok for me

File: web/config/proxy.ts [hunk 1]
```diff
@@ -14,11 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const serveUrlMap = {
```
Comment (juzhiyuan) [inline web/config/proxy.ts:17]: serverURLXX?

**Review by juzhiyuan — CHANGES_REQUESTED** [review]

File: web/config/proxy.ts (hunk 1 above)
Comment (LiteSun) [reply web/config/proxy.ts:17]: > serverURLXX?

refer to https://pro.ant.design/blog/proxy-cn#%E9%AB%98%E7%BA%A7%E7%94%A8%E6%B3%95

**Comment by membphis** [comment]
CI failed

File: web/config/proxy.ts (hunk 1 above)
Comment (juzhiyuan) [reply web/config/proxy.ts:17]: ok

**Comment by LiteSun** [comment]
> CI failed
fixed.

**Comment by LiteSun** [comment]
> > CI failed

fixed.

**Review by juzhiyuan — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]
LGTM

**Comment by LiteSun** [comment]
cc @idbeta

**Comment by idbeta** [comment]
> cc @idbeta

Thanks for inform

**Pull request merged**